pkg/distributor/distributor.go

```diff
@@ -2034,7 +2038,7 @@ func (cm *labelValuesCardinalityConcurrentMap) toResponse(isMultiZone bool, repl
 			total := uint64(0)
 
 			for _, countByZone := range dataByReplicationSet {
-				total += approximateFromZones(isMultiZone, replicationFactor, countByZone)
+				total += approximateFromZones(isMultiZone, zonesTotal, replicationFactor, countByZone)
```

Comment:
Similar to the comment left on UserStats(), when the ingest storage is enabled here we simply want to get the max of each replication set.